providers/databricks/src/airflow/providers/databricks/plugins/databricks_workflow.py

```diff
@@ -387,7 +387,7 @@ def get_link(
         return url_for("RepairDatabricksTasks.repair", **query_params)
 
 
-class RepairDatabricksTasks(AirflowBaseView, LoggingMixin):
+class RepairDatabricksTasks(BaseView, LoggingMixin):
```

Comment:
Hmm, should this extend the FAB AirflowBaseView, or be moved to a normal flask blueprint instead? Either way, some compat work with AF2 needs to be done here I think.